packages/datetime/src/dateInput.tsx

```diff
@@ -232,6 +239,7 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
                     timePickerProps={{ ...this.props.timePickerProps, precision: this.props.timePrecision }}
                 />
             );
+        const wrappedPopoverContent = <div ref={this.setContentRef}>{popoverContent}</div>;
```

Comment:
another wrapper element 😢 
guess that's how it is.

Reply:
- [ ] acknowledge sadness 😿

Reply:
😂  feeling better already